### PR TITLE
Used triple quote instead of  \"

### PR DESCRIPTION
**Pull request by SanjaySargam**
## Purpose / Description
Used triple quotes

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by criticalAY — APPROVED** [review]
LGTM

**Comment by SanjaySargam** [comment]
image
Is this test failed due to changes that I have done?

**Comment by criticalAY** [comment]
Its a flaky one I guess the second approver is requested to re run the tests

**Comment by david-allison** [comment]
Been a while, closing and reopening to bump tests

File: AnkiDroid/src/main/java/com/ichi2/libanki/Decks.kt [hunk 1]
```diff
@@ -993,77 +993,77 @@ class Decks(private val col: Collection) : DeckManager() {
...
-                "\"usn\": 0," +
-                "\"desc\": \"\"," +
-                "\"dyn\": 0," + // anki uses int/bool interchangeably here
-                "\"collapsed\": false," +
-                "\"browserCollapsed\": false," + // added in beta11
-                "\"extendNew\": 0," +
-                "\"extendRev\": 0" +
+                """"newToday": [0, 0],""" + // currentDay, count
+                """"revToday": [0, 0],""" +
+                """"lrnToday": [0, 0],""" +
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Decks.kt:1002]: This is worse. Are there alternate strategies?

**Pull request closed without merging**